### PR TITLE
Align cards style (alt)

### PR DESCRIPTION
**Pull request by taniki**
CF opendatateam/udata#1383

Same as #228 but keep the carousel

### CODE REVIEW

File: theme/less/gouvfr-dataset.less [hunk 1]
```diff
@@ -210,68 +210,6 @@ body.dataset-display {
         }
     }
 
-    .reuses-list.in-modal {
```
Comment (abulte) [inline theme/less/gouvfr-dataset.less]: cf https://github.com/etalab/udata-gouvfr/pull/228#pullrequestreview-73756697, I don't think those styles should be removed.

**Comment by taniki** [comment]
keeping the carousel but preparing for card only view

screenshot-2018-1-25 home - udata

**Comment by taniki** [comment]
image

File: theme/less/gouvfr/community.less [hunk 2]
```diff
@@ -124,152 +103,6 @@
         font-size: 13px;
     }
 
-    .reuses-list {
```
Comment (abulte) [inline theme/less/gouvfr/community.less]: Check if not used anywhere, or moved to `udata` core. If not, this must not be removed I think.

**Comment by abulte** [comment]
The last screenshot is quite neat 👍 

Food for thoughts:
- I'm disturbed by the white space right of the carousel
- Maybe we should have a more compact (height) style for the reuses card under the carousel

**Pull request merged**